src/components/shared/card.js

```diff
@@ -384,12 +426,12 @@ export function ArmorCard(props) {
     )
 }
 
-ArmorCard.propTypes={
+ArmorCard.propTypes = {
```

Comment:
This has really necessitated a style file for our IDEs. We should set up ESLINT. I think that should fix this but not certain.